docs/runbooks.md

```diff
@@ -0,0 +1,147 @@
+# Rollout-operator runbooks
+
+## Helm chart
+
+A Helm chart is available [here](https://github.com/grafana/helm-charts/tree/main/charts/rollout-operator) for the `rollout-operator`.
+
+## Alerts
+
+### IncorrectWebhookConfigurationFailurePolicy
+
+This alert fires when one of the validating webhooks used by the `rollout-operator` has its failure policy set to `Ignore`.
+
+In normal operations the failure policy should be set to `Fail`.
+
+```
+kind: ValidatingWebhookConfiguration
+...
+  failurePolicy: Fail
+```
+
+How it **works**:
+
+- This alert checks on the configured failure policy of the `pod-eviction` and `zpdb-validation` validating webhooks via the `kube_validating_webhook_failure_policy` metric
+- Although it may be valid to temporarily enable an `Ignore` failure mode, normal operations should have the failure mode set to `Fail`
+- When in `Ignore` mode the Kubernetes API server ignores a webhook failure if the webhook can not be reached
```

Comment:
```suggestion
- When in `Ignore` mode the Kubernetes API server ignores a webhook failure if the webhook can not be reached or if it rejects the request
```

Reply:
Ah, I thought it also meant the rejection was ignored - then in that case, your suggestion sounds good to me.